LayoutTests/editing/pasteboard/copy-text-from-uneditable-element.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<script src="../../resources/js-test.js"></script>
+<script src="../../resources/ui-helper.js"></script>
+<button id=button>Copy foo</button>
+<input id="result">
+<script>
+jsTestIsAsync = true;
+
+addEventListener("load", async () => {
+    button.addEventListener("click", () => {
+        const input = document.createElement("input");
+        input.hidden = true;
+        input.value = "value";
+        document.body.append(input);
+        input.select();
+        document.execCommand("copy");
+        input.remove();
+    });
+    document.addEventListener("copy", (e) => {
+        e.clipboardData?.clearData();
+        e.clipboardData?.setData("text/plain", "foo");
+        e.preventDefault();
+    }, true);
+
+    doneWithPaste = false;
+    result.addEventListener("paste", event => {
+        doneWithPaste = true;
+    });
+
+
+    if (window.testRunner) {
+        await UIHelper.activateElement(button);
+        result.focus();
+        document.execCommand("Paste");
+        await shouldBecomeEqual("doneWithPaste", "true");
+        shouldBeEqualToString("result.value", "foo");
```

Comment:
We should move this to the paste event listener so that the validation happens when we run this test in a browser.